ui/app/AppLayouts/CommunitiesPortal/controls/CommunityTagsPicker.qml

```diff
@@ -19,7 +19,7 @@ ColumnLayout {
 
     onSelectedTagsChanged: {
         var obj = JSON.parse(tags);
-        var array = JSON.parse(selectedTags);
+        var array = selectedTags.length ? JSON.parse(selectedTags) : [];
```

Comment:
`var` is out of style. It's cooler to use `const` or `let` (`const` in this case)

Reply:
fixed